### ISSUE TITLE
org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream

### ISSUE THREAD

**Issue by tkd4444** [jitsi/jitsi-videobridge#500]
I'm getting these warnings on the videobridge, any ideas ?

```
JVB 2017-07-14 09:55:40.473 INFO: [5250041] org.jitsi.impl.neomedia.rtp.FrameDesc.log() keyframe,stream=1148476830 ssrc=3835910941,idx=0,ts=2139879939,independent=true,min_seen=18150,max_seen=18150,start=18150,end=-1
JVB 2017-07-14 09:55:40.691 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:41.192 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:41.692 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:42.192 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:42.692 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:43.192 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:43.513 INFO: [5250041] org.jitsi.impl.neomedia.rtp.FrameDesc.log() keyframe,stream=1148476830 ssrc=3835910941,idx=0,ts=2140151379,independent=true,min_seen=18316,max_seen=18316,start=18316,end=-1
JVB 2017-07-14 09:55:43.692 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
JVB 2017-07-14 09:55:44.192 WARNING: [5249996] org.jitsi.impl.neomedia.rtcp.RTCPReceiverFeedbackTermination.log() no_remb,stream=2101451760
```

**Comment by gpolitis** [comment]
These warnings are harmless, if they go away after the call has run for a while (if they don't, there could be a problem). The engine needs some data before it can produce a remb.

**Comment by tkd4444** [comment]
How ling is a "while" ?

I mean in P2P mode the conference is very solid, but when we have conference handled thru the videobridge with 9+ participants some of them can't see the screenshare for example or cant see other participant video. Is there any special configuration directive to ensure maximum stability of the call ? I know it's difficult since all of the participants are connecting from different parts of the planet.

**Comment by tkd4444** [comment]
This is the config which we use, the and we are using the latest release:

```
org.jitsi.videobridge.AUTHORIZED_SOURCE_REGEXP=focus@auth.domain_name/.*
org.jitsi.videobridge.ENABLE_STATISTICS=true
org.jitsi.videobridge.STATISTICS_TRANSPORT=pubsub
org.jitsi.videobridge.PUBSUB_SERVICE=domain_name
org.jitsi.videobridge.PUBSUB_NODE=sharedStatsNode
org.ice4j.ice.harvest.ALLOWED_INTERFACES=ens3
org.ice4j.ice.harvest.STUN_MAPPING_HARVESTER_ADDRESSES=stun:stun.l.google.com:19302,stun1.l.google.com:19302,stun2.l.google.com:19302
org.jitsi.videobridge.TCP_HARVESTER_PORT=4443
org.jitsi.videobridge.rtcp.strategy=org.jitsi.impl.neomedia.rtcp.termination.strategies.BasicRTCPTerminationStrategy
org.jitsi.service.neomedia.VideoMediaStream.REQUEST_RETRANSMISSIONS=false
org.jitsi.videobridge.DISABLE_NACK_TERMINATION=true
```

**Comment by gpolitis** [comment]
These problems should have nothing to do with the specific warnings that you're seeing in the JVB logs. 

    org.jitsi.service.neomedia.VideoMediaStream.REQUEST_RETRANSMISSIONS=false
    org.jitsi.videobridge.DISABLE_NACK_TERMINATION=true

These are exactly the opposite of what you need to have and can cause the issues that you're describing :-) I'm wondering what lead you to set these values.

**Comment by tkd4444** [comment]
Sorry that was a typo, the last two were commented out.

**Comment by gpolitis** [comment]
Then it could either be because the participants don't have enough bandwidth or because we have a bug in the bridge. You can try disabling adaptivity by setting

    org.jitsi.videobridge.TRUST_BWE=false

This will make the bridge to not take into account the bandwidth estimations.

btw have you tried on https://meet.jit.si/? It would be interesting to compare

**Comment by tkd4444** [comment]
Yeah I was trying that, it's +/- the same. I'm connecting from Europe and I guess meet.jit.si servers are located in Europe too. Our environment runs on AWS in US, so there can be the catch too. Do you run the meet.jit.si with the latest nightly updates or the stable ones ?

It would be very nice to have the possibility to loadbalance a single conference between two or more videobridges. In this way we could have videobridges across different locations around the globe within AWS and the users would connect to the closest videobridge.

**Comment by tkd4444** [comment]
While keeping the communication between videobridges from site to site, relaying on a good infrastructure.

**Comment by gpolitis** [comment]
> Do you run the meet.jit.si with the latest nightly updates or the stable ones ?

Neither, it's more recent than what's in stable, but less recent than what's in unstable.

> It would be very nice to have the possibility to loadbalance a single conference between two or more videobridges.

@bgrozev is working on that.

Do you see a ninja icon next to the GSM bars for the participants that appear suspended? What exactly are you observing?

**Comment by tkd4444** [comment]
We are running the latest stable thru all components.
jicofo,jibri,jigasi,videobridge,meet,jidesha,mobile app, token authication,

Never saw any ninja icon. From time to time some participants cant see some of the other participants. But when we were streaming the whole call to youtube, on the stream all the participants could be seen.

If I do `org.jitsi.videobridge.TRUST_BWE=false ` will it disable the adaptive quality adjustment ?

**Comment by tkd4444** [comment]
If you would have time, I would gladly have a call with you thru our implemented version. Last time I was joining your every two-week conference, but I wasn't asking any questions since my impression was it's more internal call of yours.

**Comment by bgrozev** [comment]
> If you would have time, I would gladly have a call with you thru our implemented version. Last time I was joining your every two-week conference, but I wasn't asking any questions since my impression was it's more internal call of yours.

Sorry, I just got to this and the community call was earlier today. The call is aimed towards the community, and we try to record it so that it is accessible to more people. It might seem internal, because a big chunk is the core team talking about what we've been cooking, but it is definitely open to the community. Questions are always welcome, but if it is with regard to debugging a specific issue that might not be the best place for discussion.
